### PR TITLE
Fixed ALLL of Craigs mistakes

### PR DESCRIPTION
**Pull request by tpo-tcmurphy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+
+- Vodka
+- Ginger beer
+- Squeeze of lime
+- Ice
+
+## Pineapple Upside Down
+
+- 1 oz vodka
+- 1 oz amaretto
```
Comment (tpo-dpassare) [inline README.md:15]: @tpo-tcmurphy Hey bro, do you think that should be capitalized?

**Comment by tpo-tcmurphy** [comment]
Did you get this??

File: README.md (hunk 1 above)
Comment (tpo-kgoguen) [reply README.md:15]: one more comment.

Comment (tpo-tcmurphy) [reply README.md:15]: Yes I do....made the changes

Comment (tpo-cmaples) [reply README.md:15]: agreed it should definitleyabua;n be capiatialiazed

Comment (tpo-kgoguen) [reply README.md:15]: Query

**Comment by tpo-dpassare** [comment]
LGTM

**Pull request merged**